lib/main.coffee

```diff
@@ -189,6 +191,13 @@ module.exports = Hydrogen =
             @kernelManager.restartRunningKernelFor grammar, (kernel) =>
                 @onKernelChanged kernel
 
+        else if command is 'shutdown-kernel'
+            @clearResultBubbles()
+            # Note that destroy alone does not shut down a WSKernel
+            kernel.shutdown()
+            @kernelManager.destroyRunningKernelFor grammar
```

Comment:
Is it possible to modify `@kernelManager.destroyRunningKernelFor` to work for `WSKernels` as well?